packages/client/tests/e2e/nextjs-schema-not-found/_shared/test.ts

```diff
@@ -1,30 +1,70 @@
 import { $ } from 'zx'
 
 /**
- * Starts the Next.js server and tests the endpoint
- * @param endpoint The endpoint to test
+ * Starts the Next.js server and tests the endpoint. It tests:
+ * - No workaround + Server Components: if fails, nice error message
+ * - No workaround + non-Server Components: if fails, nice error message
+ * - Workaround + Server Components: should succeed
+ * - Workaround + non-Server Components: should succeed
+ * @param endpoint the endpoint to test
+ * @param serverComponents whether we use server components or not
  */
-async function test(endpoint: string) {
-  await $`rm -fr .next/standalone/node_modules/next`
-  const nextJsProcess = $`node .next/standalone/server.js`
+async function test(endpoint: string, serverComponents: boolean) {
+  console.log(`Testing ${endpoint} with WORKAROUND=${process.env.WORKAROUND}`)
 
+  // prepare and start the next.js server
+  const nextJsBuild = await $`pnpm exec next build`.nothrow()
+  await $`rm -fr .next/standalone/node_modules/next`.nothrow()
```

Comment:
This is a workaround for an existing Next.js bug, right? We should link there to explicitly document that here.

Reply:
I guess maybe I should try to start it this way https://nextjs.org/docs/api-reference/cli#production